packages/plugin-vue/src/style.ts

```diff
@@ -36,8 +45,19 @@ export async function transformStyle(
     return null
   }
 
+  // version property of result.map is declared as string
+  // but actually it is a number
+  type RawSourceMap = Omit<Exclude<typeof result.map, undefined>, 'version'>
```

Comment:
I think we should move this type to the top-level so it's consistent with the rest of the codebase.